Examples/Scripts/PreFreeSurferPipelineBatch.sh

```diff
@@ -252,9 +252,18 @@ main()
 		#     Average any repeats and use Spin Echo Field Maps for readout
 		#     distortion correction
 		#
-		#   "GeneralElectricFieldMap"
-		#     Average any repeats and use General Electric specific Gradient
-		#     Echo Field Map for readout distortion correction
+		#   "GEHealthCareLegacyFieldMap"
```

Comment:
Are we removing "GeneralElectricFieldMap" entirely as an option?  Is there any reason that we can't/shouldn't continue to support that as an alternative option name for backwards compatibility?

Reply:
I think it was left in the actual parsing code.

Reply:
Maybe I missed it, but I don't think so...

Reply:
Ok.  Given the need to now supply `DeltaTE`, it seems reasonable to no longer support the old `GeneralElectricFieldMap` as an option, and good that we renamed it (as you just did).